Prog/Q1 and Q2 recode.do

```diff
@@ -0,0 +1,124 @@
+
+clear
+set more off
+
+*log using "L:\Libraries\Voices\Prog\survey data table generation.log", replace 
+log using "D:\DCDATA\Libraries\Voices\Prog\Q1 and Q2 recode.log", replace 
+
+/* *****************************************************************************
+* 
+* Program: Q1 and Q2 recode.do
+* Library: Voices 
+* Project: VoicesDMV 2017
+* Author: CARL HEDMAN
+* Created: 10/6/2017
+* Version: Stata/MP 15
+* Environment: Windows:
+* Description: 
+*
+* Modifications: 
+*/
+**Use raw open eneded file
+use "L:\Libraries\Voices\Raw\Q1.DTA", clear
+rename CaseID caseid
+
+**Merge recode dataset
```

Comment:
I would add a comment to the program here to indicate that this data set comes from the Excel file that contains your manual recoding.